### DISCUSSION TITLE
create extension isn't working as expected

### DISCUSSION THREAD

**Discussion by tsschulz72** [supabase/supabase#13918] [Questions]
Hi, I tried to install pg_cron in both ways. First by package, and then manual (from gitub source). Both seemed to be ok. I also made "create extension pg_cron" successful. But after that, I have only the table "job" in the scheme cron.

I drop pg_cron multiple times, what was also successful, recreated pg_cron - and have ever only that table. When I put jobs in it, they also don't start.

What did I wrong?

**Comment by GaryAustin1** [comment]
This seems more appropriate as a discussion or for a Discord session.  What you are asking about is all Postgres, assuming you are doing this on a local instance (it would not work on a Hosted instance).  Supabase also asks that you check that you have gone to Github discussions and Discord to see if you can find answers and mark that you did so when declaring a bug.

**Reply by tsschulz72** [reply]
Maybe :) But, even that I think, I made something wrong - an unexpected result often is a bug, so it's not that wrong to declare it as a bug. Or am I wrong with that?
